lib/build/cpp/cxx

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env sh
+set -e
+
+exec ~/.symbolica/build/cc -nostdinc++ "$@"
```

Comment:
Do you think we should ultimately be calling through to `clang++` here instead of `clang-12` because from what I've read `clang++` will set a few other C++ specific flags such as `-lc++` when linking.